### ISSUE TITLE
Fix the following issues which makes the build 'dirty'

### ISSUE THREAD

**Issue by lolski** [graknlabs/grakn#4803]
1. : lib.bzl is deprecated
```
DEBUG: /private/var/tmp/_bazel_haikalpribadi/3cdeaa1301f056ec7b4e7ee46bda5c79/external/bazel_skylib/lib.bzl:30:1: WARNING: lib.bzl is deprecated and will go away in the future, please directly load the bzl file(s) of the module(s) needed as it is more efficient.
```

2. Attempting to add duplicate files 
```
INFO: From PackageTar console/grakn-core-console.tar.gz:
Duplicate file in archive: ./console/services/lib/US_export_policy.jar, picking first occurrence
Duplicate file in archive: ./console/services/lib/local_policy.jar, picking first occurrence
INFO: From PackageTar server/grakn-core-server.tar.gz:
Duplicate file in archive: ./server/services/lib/US_export_policy.jar, picking first occurrence
Duplicate file in archive: ./server/services/lib/local_policy.jar, picking first occurrence
```

**Comment by haikalpribadi** [comment]
This has been labelled `in progress` for a while now. What's the status of this issue @vmax ?

**Comment by vmax** [comment]
@haikalpribadi I believe it was fixed by graknlabs/bazel-distribution#63
